### PR TITLE
Use bips for slippage

### PR DESCRIPTION
**Pull request by anxolin**
Continuation on https://github.com/cowprotocol/app-data/pull/53

It makes use of the new `bip` definition to redefine the quote.

This is a breaking change for the metadata `quote`, so I bumped the version of the schema and the metadata to a new major version `v1.0.0`

Basically, we want to make it contain an integer instead of a string.
`quote: { slippageBips: "50" }`

will become:
`quote: { slippageBips: 50 }`

I also added tests for these changes.

### CODE REVIEW

**Comment by anxolin** [comment]
@MartinquaXD i just tagged you in case this breaking changes affects you

**Comment by MartinquaXD** [comment]
The backend does not parse this field so this will not cause problems on our end. 👌

File: test/schema.spec.ts [hunk 1]
```diff
@@ -628,7 +629,7 @@ describe('Schema v0.11.0', () => {
   )
 })
 
-describe('Schema v0.12.0', () => {
+describe('Schema v0.12.0: Add partner fee', () => {
```
Comment (alfetopito) [inline test/schema.spec.ts:632]: Version 0.12 doesn't belong to this PR, does it?

In fact, with this version bump there won't be a 0.12 version.

**Review by alfetopito — APPROVED** [review]
Approved with a nitpick comment

File: test/schema.spec.ts (hunk 1 above)
Comment (anxolin) [reply test/schema.spec.ts:632]: Well, I created 0.12.0 in the previous PR, I just didn't release it. Happy to delete 0.12.0 if we prefer to not create it at all.

Comment (anxolin) [reply test/schema.spec.ts]: a4789f02b2c2e7a852ee52febacb1e2368c45428
